templates/cli/lib/commands/init.js.twig

```diff
@@ -269,6 +269,7 @@ const init = new Command("init")
     .option("--organizationId <organizationId>", "{{ spec.title|caseUcfirst }} organization ID")
     .option("--projectId <projectId>", "{{ spec.title|caseUcfirst }} project ID")
     .option("--projectName <projectName>", "{{ spec.title|caseUcfirst }} project ID")
+    .option(`-sh, --self-hosted`, `Flag for enabling custom endpoint for self hosted instances for non-logged in users`)
```

Comment:
what are we trying to achieve here?

Reply:
In case the user is not logged in, the project init will prompt the user to login,
By default, all logins are  targeted at the cloud, this will give the user the option to add an endpoint

Reply:
No lets not do this.. There is already a dedicated command for setting endpoint with appwrite client. 
We should make sure every command has one job and one job only